base/message/schema/verifiable-presentation-schema.json

```diff
@@ -0,0 +1,64 @@
+{
+    "$schema": "http://json-schema.org/draft-07/schema#",
+    "title": "Verifiable Presentation",
+    "type": "object",
+    "properties": {
+        "id": {
+            "type": "string",
+            "description": "Presentation identifier"
+        },
+        "type": {
+            "type": "array",
+            "items": {
+                "type": "string"
+            },
+            "description": "Types of the presentation"
+        },
+        "verifiableCredential": {
+            "type": "array",
+            "items": {
+                "type": "string"
+            },
+            "description": "Array of verifiable credentials included in the presentation"
+        },
+        "proof": {
+            "type": "object",
+            "properties": {
+                "type": {
+                    "type": "string",
+                    "description": "Proof type"
+                },
+                "created": {
+                    "type": "string",
+                    "format": "date-time",
+                    "description": "Proof creation time"
+                },
+                "proofPurpose": {
+                    "type": "string",
+                    "description": "Purpose of the proof"
+                },
+                "verificationMethod": {
+                    "type": "string",
+                    "description": "Verification method URI"
+                },
+                "jws": {
+                    "type": "string",
+                    "description": "JSON Web Signature"
+                }
+            },
+            "required": [
+                "type",
+                "created",
+                "proofPurpose",
+                "verificationMethod",
+                "jws"
+            ]
+        }
+    },
+    "required": [
+        "id",
+        "type",
+        "verifiableCredential",
+        "proof"
```

Comment:
if we decide that we need the VP schema in the DCP spec, the `proof` still shouldn't be mandatory and maybe not even part of the schema at all.

Reply:
done